layouts/shortcodes/kubeweekly.html

```diff
@@ -0,0 +1,18 @@
+{{- $id := .Get "id" | default (printf "kubeweekly-%d" .Ordinal) -}}
+<section class="section-feature" id="{{ $id }}">
+    <div class="kubeweekly-inner">
+    <div id="mc_embed_signup">
+        <form action="https://kubeweekly.us10.list-manage.com/subscribe/post?u=3885586f8f1175194017967d6&amp;id=11c1b8bcb2" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
+            <div id="mc_embed_signup_scroll">
+                <p>{{ T "main_kubeweekly_baseline" }}</p>
+
+                <input type="email" value="" name="EMAIL" class="email" id="mce-EMAIL" placeholder="{{ T "input_placeholder_email_address" }}" aria-label="email" required>
+            <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
+            <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_3885586f8f1175194017967d6_11c1b8bcb2" tabindex="-1" value=""></div>
+            <div class="clear"><input type="submit" value="{{ T "subscribe_button" }}" name="subscribe" id="mc-embedded-subscribe" class="button"></div>
+            </div>
+        </form>
+        <h5><a class="kubeweekly-signup" href="https://us10.campaign-archive.com/home/?u=3885586f8f1175194017967d6&id=11c1b8bcb2" aria-label="Kube Weekly">{{ T "main_kubeweekly_past_link" }}</a></h5>
```

Comment:
This tag is a bit weird as `main_kubeweekly_past_link` doesn't exist. `main_kubernetes_past_link` does exist and it seems PR #12873 was the one which introduced the typo. We might want to fix that here. My preference is to change the `en.toml` from `main_kubernetes_past_link` to `main_kubeweekly_past_link` as that's most appropriate IMO.

Moreover, once that change is introduced, you will notice that the style for the text is different. We will need to style the `h5` or the `a` blocks appropriately.

On the left is the live and on the right is the PR
![image](https://github.com/user-attachments/assets/2374be1d-c497-4785-8570-00a2a8a9ed23)

Reply:
Please file an issue about that.

Reply:
#49164 created